docs/content/guides/navigation/keyboard-shortcuts.md

```diff
@@ -0,0 +1,279 @@
+---
+id: 5w6juytv
+title: Keyboard shortcuts
+metaTitle: Keyboard shortcuts - JavaScript Data Grid | Handsontable
+description: Access all Handsontable features using just your keyboard. Use shortcuts you know from Google Sheets or Microsoft Excel.
+permalink: /keyboard-shortcuts
+canonicalUrl: /keyboard-shortcuts
+tags:
+  - key bindings
+  - keymap
+  - key mapping
+  - keyboard navigation
+  - hotkey
+  - accessibility
+  - function key
+  - commands
+react:
+  id: ddjw4zt8
+  metaTitle: Keyboard shortcuts - React Data Grid | Handsontable
+searchCategory: Guides
+---
+
+# Keyboard shortcuts
+
+Access all Handsontable features using just your keyboard. Use shortcuts you know from Google Sheets or Microsoft Excel.
+
+## Overview
+
+[[toc]]
+
+This page lists all of Handsontable's default keyboard shortcuts.
+
+You can easily check if a keyboard shortcut's action is compatible with Microsoft Excel or Google Sheets:
+
+- Compatible: &check;
+- Not compatible: &cross;
+
+::: tip
+
+Handsontable doesn't detect data series (doesn't differentiate between blank cells and non-blank cells). For this reason, some keyboard shortcuts (e.g.,
```

Comment:
Hello, I propose the following version for consideration:

"Handsontable neither detects data series nor differentiates blank cells and non-blank cells. Therefore, some keyboard shortcuts (such as <kbd>**Cmd**</kbd> + Arrow keys) may behave differently in particular situations than in the Microsoft Excel or Google Sheets."